src/main/java/gift/entity/Member.java

```diff
@@ -3,24 +3,31 @@
 import jakarta.persistence.*;
 import jakarta.validation.constraints.NotNull;
 
+import java.util.List;
+
 @Entity
 public class Member {
 
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    Long id;
+    private Long id;
+
     @Column(unique = true)
     @NotNull
-    String email;
+    private String email;
+
     @NotNull
-    String password;
+    private String password;
+
+    @OneToMany(mappedBy = "member")
+    private List<Wish> wishes;
```

Comment:
컬렉션의 경우 NPE를 방지하기 위해 생성까지 해주는 편이에요~
```suggestion
    private List<Wish> wishes = new ArrayList();
```

Reply:
Null Pointer Exception 방지법 감사합니다!